tests/embedding/test_laplacian_spectral_embedding.py

```diff
@@ -29,6 +33,9 @@ def test_laplacian_embedding(self):
         self.assertListEqual(expected_label, labels)
 
     def test_laplacian_embedding_elbowcut_none(self):
+        if sys.platform.startswith('darwin'):
+            pytest.skip('Test not supported on Mac OS')
```

Comment:
Let's also add an issue to track this; we may want to create a separate repo for just this, try to isolate the scikit-learn svd problem to just the code that reproduces it with their package on a github action and show that to them.

Reply:
Issue #35 has been created.